DirectProgramming/DPC++FPGA/include/streaming_qrd.hpp

```diff
@@ -42,10 +42,13 @@ template <typename T,        // The datatype for the computation
                              // elements from the pipe with each read
           typename QOut,     // Q matrix output pipe, send pipe_size
                              // elements to the pipe with each write
-          typename ROut      // R matrix output pipe, send pipe_size
+          typename ROut,     // R matrix output pipe, send pipe_size
                              // elements to the pipe with each write.
                              // Only upper-right elements of R are
                              // sent in row order, starting with row 0.
+          bool input_matrix_order = true  // Defualt value is true for standard matrix input reads (reads the matrix one column at a time).
+                                           // Flase if read order by rows (sweeps the rows by pipe size). Each read contains pipe_size samples from the same column, then
+                                            // the next read contains samples from the next column.
```

Comment:
This name is not descriptive.  How about 'k_colum_order' (k is a coding convention to indicate it's a compile time constant).

Comments extend past 80 characters - did you run clang format?